frontend/packages/kubevirt-plugin/integration-tests/tests/non-admin.scenario.ts

```diff
@@ -0,0 +1,137 @@
+import { execSync } from 'child_process';
+import { $, $$, browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '../../../../integration-tests/protractor.conf';
+import { click } from '../../../console-shared/src/test-utils/utils';
+import * as loginView from '../../../../integration-tests/views/login.view';
+import * as crudView from '../../../../integration-tests/views/crud.view';
+import * as yamlView from '../../../../integration-tests/views/yaml.view';
+import { Wizard } from './models/wizard';
+import {
+  KUBEADMIN_IDP,
+  KUBEADMIN_USERNAME,
+  BRIDGE_KUBEADMIN_PASSWORD,
+  BRIDGE_HTPASSWD_IDP,
+  BRIDGE_HTPASSWD_USERNAME,
+  BRIDGE_HTPASSWD_PASSWORD,
+} from './utils/consts';
+
+describe('Kubevirt nonadmin ui accessibilities', () => {
+  const nonAdminNS = `${testName}-nonadmin`;
+  const BROWSER_TIMEOUT = 15000;
+  const ACCESS_DENIED_ICON = $('.cos-status-box__access-denied-icon');
+
+  const verifyPermission = async function(ns: string, path: string, perm: string) {
+    const wizard = new Wizard();
+
+    // Verify permissions on VM page
+    await browser.get(`${appHost}/k8s/ns/${ns}/${path}`);
+    if (perm === 'noAccess') {
+      await browser.wait(until.presenceOf(crudView.errorMessage));
+      await browser.wait(until.presenceOf(ACCESS_DENIED_ICON));
+      // TODO: uncomment below when bz1761041 is fixed.
```

Comment:
I think I have commented on this already. I'm not a fan of having test cases silently passing on something that is actually not being tested. I would prefer to have the tests to have a structure that allows skipping testing some of the functionality by using `xit`. Especially in case like this where it takes so long time to fix.
According to Tomas's comment in the BZ 1761041, it's unlikely user would encounter such issue under normal circumstances, is it beneficial to have such specific test in upstream at all? @suomiy @jelkosz